### ISSUE TITLE
query summary functions

### ISSUE THREAD

**Issue by dwaggott** [samtools/bcftools#724]
I'm trying to extract some variant allele fraction statistics from multisample vcf n~300. I'm getting these through extracting the the REF and ALT allele counts from the FORMAT tag AD and then doing the calculations outside of bcftools. To avoid this it would be helpful if one could apply similar arithmetic operators and summary functions in the `query` command as in the `expressions`

bcftools  query -f '[\t%AD{1}/(%AD{1}+%AD{0})]\n' tmp.vcf
bcftools query -f 'sum(%AD{1})/(sum(%AD{0})+sum(%AD{1}))' tmp.vcf

**Comment by pd3** [comment]
I agree this would be useful for some uses and considered adding this a while ago, but at the end decided against it. Mainly because the complexity of using this feature and implementing it seemed bigger than writing a small dedicated script for post-processing the raw `query` output.
